src/main/webapp/app/shared/metis/metis.util.ts

```diff
@@ -113,6 +113,7 @@ export enum ReferenceType {
     QUIZ = 'quiz',
     TEXT = 'text',
     FILE_UPLOAD = 'file-upload',
+    USER = 'USER',
```

Comment:
This is inconsistent with the other enum values.

Reply:
The alternative reference types, such as for Post, Lecture, etc., are also capitalized (You can see it if you expand the shown diff). Only exercise types are in lowercase. Consequently, I decided to use all uppercase letters for User to make it consistent with Post, Lecture, ...